io.catenax.bom_as_specified/3.0.0/BomAsSpecified.ttl

```diff
@@ -0,0 +1,204 @@
+    samm:description "Entity encapsulating the properies describing a part"@en ;
+    samm:properties (:ownerPartId [
+  samm:property :partVersion ;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partQuantity ;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partDescription ;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partClassification ;
+  samm:optional "true"^^xsd:boolean
+]) .
+
+:ownerPartId a samm:Property ;
+    samm:preferredName "owner part ID"@en ;
+    samm:description "This is the key field of the component which usually keeps the part numbers used in after-sales, e.g. when repairing broken parts and searching for a replacement. This ownerPartId itself isn't usually bound to one part version, with the assumption that all part versions with the same ownerPartId are mutually interchangeable."@en ;
+    samm:characteristic samm-c:Text ;
+    samm:exampleValue "5A1BA91" .
+
+:partVersion a samm:Property ;
+    samm:preferredName "part version"@en ;
+    samm:description "This is the version of the part. The engineering will at times supercede an older part version by a newer one, which might have different material aspects, physical dimensions etc., still maintaining compatibility."@en ;
+    samm:characteristic samm-c:Text ;
+    samm:exampleValue "05" .
+
+:partQuantity a samm:Property ;
+    samm:preferredName "part quantity"@en ;
+    samm:description "This is the quantity how often this part is in the item."@en ;
+    samm:characteristic :PartQuantityCharacteristic .
+
+:partDescription a samm:Property ;
+    samm:preferredName "part description"@en ;
+    samm:description "The description of the part in the primary language of the production facility of the product owner."@en ;
+    samm:characteristic samm-c:Text .
+
+:partClassification a samm:Property ;
+    samm:preferredName "part classification"@en ;
+    samm:description "The part classification."@en ;
+    samm:characteristic :PartClassificationCharacteristic .
+
+:PartQuantityCharacteristic a samm:Characteristic ;
+    samm:preferredName "part quantity characteristic"@en ;
+    samm:description "The characteristic of the part quantity."@en ;
+    samm:dataType :PartQuantityEntity .
+
+:PartClassificationCharacteristic a samm-c:Set ;
+    samm:preferredName "part classification characteristic"@en ;
+    samm:description "The characteristic of the part classification."@en ;
+    samm:dataType :PartClassificationEntity .
+
+:PartQuantityEntity a samm:Entity ;
+    samm:preferredName "part quantity entity"@en ;
+    samm:description "The entity encapsulating the properties describing the quantity of a part."@en ;
+    samm:properties (:measurementUnit :quantityNumber) .
+
+:measurementUnit a samm:Property ;
+    samm:preferredName "measurement unit"@en ;
+    samm:description "Unit of measurement for the quantity of serialized objects."@en ;
+    samm:characteristic samm-c:UnitReference ;
+    samm:exampleValue "litre"^^samm:curie .
```

Comment:
example value should be "unit:litre" (without samm:curie), else it is not valid according to json schema.